examples/controls_example/public/react_controls/control_group/types.ts

```diff
@@ -54,6 +61,8 @@ export type ControlGroupApi = PresentationContainer &
   PublishesTimeslice &
```

Comment:
Maybe out of scope for this PR, but should we expand the `ControlGroupApi` in the example to optionally include the `reportInvalidSelections` method? In production, this method may display a popover indicating the validation changes introduced in https://github.com/elastic/kibana/pull/174201. Then the child embeddables can call the parent method as needed.

Reply:
There have been conversations between @ThomThomson and @teresaalvarezsoler about removing the controls validation tour that would negate the need for a control to bubble invalid status up to the control group.